docs/source/how-to/understand-keys-and-how-to-change-them.md

```diff
@@ -0,0 +1,146 @@
+dsets = catalog_subset.to_dataset_dict()
+print(dsets)
+```
+
+### Investigating the keys
+
+The keys for these datasets include some helpful information - but you might be wondering what this all means and where this text comes from...
+
+```{code-cell} ipython3
+print(list(dsets))
+```
+
+When `intake-esm` aggregates these datasets, it uses some pre-determined metadata, defined in the catalog file. We can look at which fields are used for aggregation, or merging of the datasets, using the following
+
+```{code-cell} ipython3
+print(catalog.esmcat.aggregation_control.groupby_attrs)
+```
+
+Let's go back to our data catalog... and find these fields. You'll notice they are all column labels! These are key components of the metadata.
+
+```{code-cell} ipython3
+catalog_subset.df
+```
+
+## Using keys_info()
+
+These groupby attributes are columns in our catalog! This means that the datasets which will be merged using the hierarchy:
+
+```
+activity_id --> institution_id --> source_id --> experiment_id --> table_id --> grid_label
+```
+
+A more clear of taking a look at these aggregation variables using the `.keys_info()` method for the catalog:
+
+```{code-cell} ipython3
+catalog_subset.keys_info()
+```
+
+## Change our groupby/aggregation controls
+
+If we wanted to instead aggregate our datasets at the member_id level, we can change that using the following method:
+
+```{code-cell} ipython3
+original_groupby_attributes = catalog.esmcat.aggregation_control.groupby_attrs
+new_groupby_attributes = original_groupby_attributes + ["member_id"]
+print(new_groupby_attributes)
+```
+
+Now that we have our new groupby attributes, we can assign these to our catalog subset.
+
+```{code-cell} ipython3
+catalog_subset.esmcat.aggregation_control.groupby_attrs = new_groupby_attributes
+```
+
+Let's check our new keys! You'll notice we now have 97 keys, aggregated on
+
+```
+activity_id --> institution_id --> source_id --> experiment_id --> table_id --> grid_label --> member_id
+```
+
+```{code-cell} ipython3
+catalog_subset.keys_info()
+```
+
+### Load our datasets with the new keys
+
+We can now load our new datasets to our dictionary of datasets using:
+
+```{code-cell} ipython3
+dsets = catalog_subset.to_dataset_dict()
+```
+
+And if we only wanted the first key, we could use the following to grab the first key in the list. Notice how we now have our member_id at the end!
+
+```{code-cell} ipython3
+first_key = catalog_subset.keys()[0]
+first_key
+```
+
+And the .to_dask() method to load our dataset into our notebook.
+
+```{code-cell} ipython3
+ds = catalog_subset[first_key].to_dask()
+ds
+```
+
+### Compare this dataset with the original catalog configuration
+
+Compare this to our original catalog, which aggregated one level higher, placing all of the `member_id`s into the same dataset.
+
+**Notice how our metadata now mentions there are 65 member_ids in this dataset, compared to 1 in the previous dataset**
```

Comment:
````suggestion
```{note}
Notice how our metadata now mentions there are 65 member_ids in this dataset, compared to 1 in the previous dataset
```
````